openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -453,16 +463,19 @@ function processIndicators(layerAttributes, projectDef) {
                 la[ix].newProperties['SVI'] = (SVI[key]).toFixed(5);
             }
         }
-        for (var key in RI) {
-            if (key == la[ix].properties[selectedRegion]) {
-                la[ix].newProperties['IR'] = (RI[key]).toFixed(5);
+
+        if (riskIndicators !== undefined) {
```

Comment:
I am not 100% sure, but I think the former is the safest one and they are not completely equivalent